src/components/components/footer/footer.js

```diff
@@ -1,159 +1,171 @@
-          <div className="footer__legal">
-            <p className="footer__licence">
-              Portions of this content are ©1998–2018 by individual mozilla.org
-              contributors. Content available under a{' '}
-              <a
-                rel="license"
-                href="https://www.mozilla.org/foundation/licensing/website-content/"
-              >
-                Creative Commons license
-              </a>
-              .
-            </p>
-            <ul className="footer__terms">
-              <li>
-                <a
-                  rel="nofollow"
-                  href="https://www.mozilla.org/privacy/websites/"
-                >
-                  Website Privacy Notice
-                </a>
-              </li>
-              <li>
+    return (
+      <div className="footer">
+        <LayoutContained className="footer__inner">
+          <nav className="footer__primary">
+            <Link
+              to="/"
+              className="footer__primary-logo"
+              activeClassName="is-active"
+            >
+              <span className="footer__ossn-logo">
+                Open Source Student Network
+              </span>
+              <span className="footer__mozilla-logo">Mozilla</span>
+            </Link>
+
+            <div className="footer__sections">
+              <MediaQuery minWidth={480} className="footer__section-item">
+                <Navigation
+                  title="Navigation"
+                  links={footerNavigationList1}
+                  className="footer__section--navigation"
+                  id="footer-section-navigation"
+                />
+                <Navigation
+                  title="Keep in touch"
+                  links={footerNavigationList2}
+                  className="footer__section--social"
+                  id="footer-section-social"
+                />
+              </MediaQuery>
+              <MediaQuery maxWidth={479} className="footer__section-item">
+                <Navigation
+                  title="Navigation"
+                  isMobile
+                  links={footerNavigationList1}
+                  className="footer__section--navigation"
+                />
+                <Navigation
+                  title="Keep in touch"
+                  isMobile
+                  links={footerNavigationList2}
+                  className="footer__section--social"
+                />
+              </MediaQuery>
+            </div>
+          </nav>
+
+          <nav className="footer__secondary">
+            <div className="footer__legal">
+              <p className="footer__licence">
+                Portions of this content are ©1998–2018 by individual
+                mozilla.org contributors. Content available under a{' '}
                 <a
-                  rel="nofollow"
-                  href="https://www.mozilla.org/privacy/websites/#cookies"
+                  rel="license"
```

Comment:
Shouldn't this include noopener and noreferrer ?

Reply:
As long as there is no target _blank noopener and noreferrer are not needed. In any case I'll add both target _blank and noopener and noreferrer.

Reply:
Yeah sorry forgot to say about target blank as well